apiCalls.js

```diff
@@ -8,3 +8,7 @@ const api = axios.create({
 export const getAllArticles = () => {
   return api.get("/articles");
 };
+
+export const getArticleById = (id) => {
+  return api.get(`/articles/${id}`);
```

Comment:
Might be a better idea to return data.article here in the api call rather than have to dot notation to the right key from within the component the function is called.